spec/models/like_spec.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../rails_helper'
+RSpec.describe Like, type: :model do
+  user = User.create(
+    Name: 'VITS',
+    Photo: 'https:123',
+    Bio: 'A software developer based in Kenya',
+    PostsCounter: 0
+  )
+
+  post = Post.create(
+    Title: 'my title',
+    Text: 'Hello World!',
+    CommentsCounter: 1,
+    LikesCounter: 0,
+    user_id: user.id
+  )
+
+  Like.create(
+    user_id: user.id,
+    post_id: post.id
+  )
+
+  context '#increment_post_likes_counter' do
+    it 'increments post likes counter by 1' do
+      expect(User.find(post.id).LikesCounter).to eq(1)
+    end
+  end
```

Comment:
- [ ] All unit tests should be passed. Kindly fix the error. :+1: 

![test1](https://user-images.githubusercontent.com/70483835/215172305-6dc1dd29-c2b6-4bab-bdf9-d3722d40adbb.png)